galeracluster/source/notificationcmd.rst

```diff
@@ -0,0 +1,119 @@
+|               | :term:`Primary Component` that is in a complete state |
+|               | and is catching up with the cluster.                  |
++---------------+-------------------------------------------------------+
+| ``Synced``    | Indicates a node that is syncrhonized with the        |
+|               | cluster.                                              |
++---------------+-------------------------------------------------------+
+| ``Error``     | Indicates that an error has occurred.  This status    |
+|               | string may provide an error code with more            |
+|               | information on what occurred.                         |
++---------------+-------------------------------------------------------+
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Members List Format
+^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _`member-list-format`:
+
+The notification command passes with the ``--member`` parameter a list containing entries for each node that is connected to the cluster component to which the node belongs.  For each entry in the list the node uses this format:
+
+.. code-block:: text
+
+   <node UUID> / <node name> / <incoming address>
+   
++------------------+---------------------------------------------------+
+| Format           | Description                                       |
++==================+===================================================+
+| Node UUID        | Refers to the unique identifier the node receives |
+|                  | from the wsrep Provider.                          |
++------------------+---------------------------------------------------+
+| Node Name        | Refers to the node name, as you define it for the |
+|                  | :ref:`wsrep_node_name <wsrep_node_name>`          |
+|                  | parameter, in the configuration file.             |
++------------------+---------------------------------------------------+
+| Incoming Address | Refers to the IP address for client connections,  |
+|                  | as set for the                                    |
+|                  | :ref:`wsrep_node_incoming_address                 |
+|                  | <wsrep_node_incoming_address>` parameter,         |
+|                  | in the configuration file.                        |
++------------------+---------------------------------------------------+
+ 
+
+----------------------------------
+Enabling the Notification Command
+----------------------------------
+.. _`enable-notification-command`:
+
+You can enable the  notification command through the :ref:`wsrep_notify_cmd <wsrep_notify_cmd>` parameter in the configuration file. 
+
+.. code-block:: ini
+
+   wsrep_notify_cmd=/path/to/wsrep_notify.sh
+
+The node then calls the script, to each change in cluster membership and node status.  You can use these status changes in configuring load balancers, raising alerts or scripting for any other situation where you need the underlying server to respond to changes to the database.
```

Comment:
"the underlying server" ? maybe "where you need your infrastructure to respond to changes to the cluster"